apps/pie-storybook/turbo.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "extends": [
+    "//"
+  ],
+  "pipeline": {
+    "dev": {
+      "cache": false,
+      "dependsOn": [
+        "^build"
```

Comment:
I thought with storybook, dev built in non prod mode, and build built in prod mode ready for deploy? So not sure they're a dependency on one another?

Reply:
The syntax here can be a bit confusing. What this is saying is that `dev` depends on the same things as `build`, i.e., run what is required for `build` (but not `build` itself) before running `dev`.

Reply:
Makes sense 👍